README.md

```diff
@@ -57,7 +57,34 @@ app
     └── DemoApplicationDataSource.json
 ```
 
-#### General
+### Supported reference syntax
+The CLI tool will understand and resolve these kind of type reference schemas during import.
+All values with one of these keys; `("type", "attributeType", "extends", "_blueprintPath_")` will be evaluated for resolution.
+
+```bash
+# DMSS
+dmss://datasource/package/entity
+dmss://datasource/package/subfolder/entity
```

Comment:
```suggestion
sys://datasource/package/entity
sys://datasource/package/subfolder/entity
```
Leave as `sys` until we have resolved the issue regarding renaming to `dmss://`

Reply:
fixed